package.json

```diff
@@ -85,7 +85,11 @@
     "lint:js": "eslint src",
     "lint:js:fix": "eslint src --fix",
     "start": "fec dev",
-    "start:msw": "MSW=TRUE fec dev",
+    "start:stage": "fec dev --clouddotEnv=stage",
+    "start:prod": "fec dev --clouddotEnv=prod",
+    "start:msw": "NODE_ENV=development MSW=TRUE fec dev",
+    "start:msw:stage": "NODE_ENV=development MSW=TRUE fec dev --clouddotEnv=stage",
+    "start:msw:prod": "NODE_ENV=development MSW=TRUE fec dev --clouddotEnv=prod",
```

Comment:
Just a note - previously the MSW in browser was allowed for stage only. I think it was based on webpack configs being split into two separate files for prod and stage so it might work also for prod with the new config

https://github.com/osbuild/image-builder-frontend/pull/2183/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L99

Reply:
I can easily remove if it doesn't make sense! Just thumb up here and I'll adapt the PR.

Reply:
Okay, let's remove `start:msw` and `start:msw:prod` for now, we can always expand the options later :) Thank you!